content/docs/intro/vs/cloud_template_transpilers.md

```diff
@@ -50,15 +51,20 @@ The transpiler approach gives you some of the benefits of Pulumi, with the follo
   underlying building block services in your target cloud, or even HashiCorp Vault, to deliver an easy experience
   with secrets management automatic best practices built-in.
 
-* Pulumi integrates with [a number of CI/CD providers]({{< relref "/docs/guides/continuous-delivery" >}}) out of the box.
+* Pulumi integrates with [a number of CI/CD providers]({{< relref "/docs/guides/continuous-delivery" >}}) and
+  source control systems out of the box, to deliver easy continuous delivery with systems you might already be using.
```

Comment:
to deliver delivery - a style thing, maybe replace "deliver" with a synonym

Reply:
How about just going with "for easy continuous delivery"?